roles/fail2ban/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: restart fail2ban
+  systemd:
+    name: fail2ban.service
+    state: restarted
+    daemon_reload: yes
+    enabled: yes
+  become: yes
```

Comment:
I'm afraid this line should be before the line 3 to become root.
Actually I failed to restart fail2ban, while by setting this line before the line 3, I succeeded.

Reply:
Order in dict of yaml configuration should not matter, I think. (at least, other roles have similar order.)
Is it an error from ansible role, or from mis-input of sudo password?